catalog/imaging/imaging-transformer-nitf/src/main/java/org/codice/alliance/transformer/nitf/common/SegmentHandler.java

```diff
@@ -104,6 +112,29 @@ private void handleTreValues(Metacard metacard, NitfAttribute attribute, Tre tre
   private Attribute populateAttribute(Metacard metacard, String attributeName, Serializable value) {
     Attribute currentAttribute = metacard.getAttribute(attributeName);
 
+    // check if country code
+    if (COUNTRY_CODE_ATTRIBUTES.contains(attributeName)) {
+      List<Serializable> alpha3CountryCodes = new ArrayList<>();
+      String[] tokenizedCountryCodes = value.toString().split(" ");
+
+      for (String countryCode : tokenizedCountryCodes) {
+        try {
+          alpha3CountryCodes.add((new Locale("", countryCode)).getISO3Country());
+        } catch (MissingResourceException mre) {
+          LOGGER.debug("Could not convert {} to ISO-3 country code.", countryCode);
```

Comment:
❓ Should we fail completely at this point? Seems like the attribute is completely malformed or incorrect if we hit at least one of these.

Reply:
Since these are security attributes, that might be a good idea. Although having no CCs in these attributes will only make the data more hidden, not more visible as far as I know

Reply:
If you are getting MissingResourceException, it is most likely because the country code was invalid to start with. Are we not reporting issues like that in any other ways then debug?